src/Raven.Server/Documents/Indexes/MapReduce/Static/MapReduceIndex.cs

```diff
@@ -30,6 +30,8 @@ public class MapReduceIndex : MapReduceIndexBase<MapReduceIndexDefinition>
 
         private readonly Dictionary<string, AnonymousObjectToBlittableMapResultsEnumerableWrapper> _enumerationWrappers = new Dictionary<string, AnonymousObjectToBlittableMapResultsEnumerableWrapper>();
 
+        public PropertyAccessor OutputReduceToCollectionPropertyAccessor;
```

Comment:
It will work the same, thought that it might be a better place than exposing a public field.